python/pfs/drp/stella/reduceArcTask.py

```diff
@@ -190,6 +183,17 @@ def run(self, expRefList, butler, wLenFile=None, lineList=None, immediate=True):
                 for j in range(wLenLines.shape[0]):
                     wLenLinesArr[j] = wLenLines[j]
                 lineListPix = drpStella.createLineList(wLenArr, wLenLinesArr)
+
+                log.logging.getLogger("pfs::drp::stella::math::createLineList").setLevel(log.DEBUG)
+                try:
+                    lineListPix = drpStella.createLineList(wLenArr, wLenLinesArr)
+                except:
```

Comment:
You should [almost] never catch all exceptions.  Even if you do, please use `except Exception as e:` rather than `sys.exc_info`.